### DISCUSSION TITLE
Emoji command

### DISCUSSION THREAD

**Discussion by ComradeTurtle** [Shadow-Creation-Team/Protector#44] [Feature Request & Ideas]
What I am suggesting is an emoji command which will take an emoji as an argument and add it to the current server. It is a feature in other bots as well and it is quite useful.

**Usage example:**
`p!emoji :kekw:`

**Comment by GeoSyOG** [comment]
good job comrade

**Comment by andreaskapsalhs** [comment]
Comrade has been awoken, how can we possibly deny this feature.

**Comment by tougrel** [comment]
I'm converting this into an issue (#57). Looking for a v0.5.1 release here
